client/Spec/__snapshots__/frontend.test.js.snap

```diff
@@ -0,0 +1,325 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
still figuring out how these snapshot tests work, but documentation says to keep them up in github too